RelhaxModpack/RelhaxModpack/Utils/Translations.cs

```diff
@@ -1343,7 +1343,7 @@ public static void LoadTranslations()
             //button for launching the editor from the main application window
             English.Add("LauchPatchDesignerText", "Launch Patch Designer");
             German.Add("LauchPatchDesignerText", "Starte Patch Designer");
-            Polish.Add("LauchPatchDesignerText", TranslationNeeded);
+            Polish.Add("LauchPatchDesignerText", "Uruchom Patch Designer");
```

Comment:
I keep it as Patch Designer since the Patch Designer itself, as a tool, is hard-coded English :D